client/src/components/Cover/PendingSupervisor/PendingSupervisor.js

```diff
@@ -31,16 +31,35 @@ class PendingSupervisor extends React.Component {
       ]
     };
   }
+
+  save() {
+    console.log("In save");
+    var json = fetch("/covrequests", {
```

Comment:
The hardcode on line 22 is unnecessary now right because we're pulling this from the backend?

Reply:
Actually I don't see /covrequests anywhere on the backend. Does this fetch request actually work?